quickwit/quickwit-metastore/src/metastore/postgresql_metastore.rs

```diff
@@ -571,46 +571,99 @@ impl Metastore for PostgresqlMetastore {
         Ok(())
     }
 
-    #[instrument(skip(self, split_metadata), fields(index_id=index_id, split_id=split_metadata.split_id))]
```

Comment:
so we will lose the logging of the `split_ids` here. Do we have it somewhere else?

split_ids are VERY useful when debugging.
Can you readd split_ids one way or another or add an issue to do it?

Reply:
Added!